c2corg_ui/templates/route/view.html

```diff
@@ -17,65 +19,162 @@
+      <h4><span translate>General</span><span class="glyphicon glyphicon-menu-right"></span></h4>
+      <div class="name-icon">
+        <span class="glyphicon glyphicon-stats"></span>
+        <p translate>General</p>
+      </div>
+      <span class="detail-text accordion">
+         % if route['route_types']:
+        <p><span translate>route_types</span>: <b>${route['route_types']}</b></p>
+         % endif
+        % if route['elevation_max']:
+        <p><span translate>elevation_max</span>: <b>${route['elevation_max']} m </b></p>
+        % endif
+        % if route['elevation_min']:
+        <p><span translate>elevation_min</span>: <b>${route['elevation_min']} m </b></p>
+        % endif
+      </span>
+    </div>
+    ## SUMMARY
     % if locale['summary']:
-      <div class="name-icon-value col-sm-6">
-        <div class="name-icon">
-          <span class="glyphicon glyphicon-list-alt"></span>
-          <span class="detail-name" translate>summary</span>
-        </div>
-        <span class="detail-text">${show_attr(locale, 'summary')}</span>
+    <div class="name-icon-value col-sm-6" ng-click="detailsCtrl.openTab($event)">
+      <h4><span translate>summary</span><span class="glyphicon glyphicon-menu-right"></span></h4>
+      <div class="name-icon">
+        <span class="glyphicon glyphicon-list-alt"></span>
+        <p translate>summary</p>
       </div>
+      <span class="detail-text accordion">
+        ${locale['summary']}
+      </span>
+    </div>
     % endif
+    ##DIFFICULTY
+    <div class="name-icon-value col-sm-6" ng-click="detailsCtrl.openTab($event)">
+      <h4><span translate>Difficulty</span><span class="glyphicon glyphicon-menu-right"></span></h4>
+      <div class="name-icon">
+        <span class="glyphicon glyphicon-alert"></span>
+        <p translate>Difficulty</p>
+      </div>
+      <span class="detail-text accordion">
+        %if route['height_diff_difficulties']:
+        <p><span translate>height_diff_difficulties</span>: <b>${route['height_diff_difficulties']} m</b></p>
+        % endif
+        % if 'equipment_rating' in route and route['equipment_rating']:
+        <p><span translate>equipment_rating</span>: <b>${route['equipment_rating']}</b></p>
+        % endif
+        % if 'engagement_rating' in route and route['engagement_rating']:
+        <p><span translate>engagement_rating</span>: <b>${route['engagement_rating']}</b></p>
+        % endif
+      </span>
+    </div>
+    ##ACCESS
+    <div class="name-icon-value col-sm-6" ng-click="detailsCtrl.openTab($event)">
+      <h4><span translate>Access</span><span class="glyphicon glyphicon-menu-right"></span></h4>
+      <div class="name-icon">
+        <span class="glyphicon glyphicon-screenshot"></span>
+        <p translate>Access</p>
+      </div>
+      <span class="detail-text accordion" ng-init="lift_access=${route['lift_access']}">
+        <span translate >Approche en remontée mécanique</span>: 
```

Comment:
Extra space after "translate" :p 
Please use an english string for the attribute title.